src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { SetTodo } from '../../interfaces/interfaces';
+
+type Props = {
+  loading: boolean;
+  handleCloseModal: () => void;
+  user: User | null;
+  choseTodo: SetTodo;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  loading,
+  handleCloseModal,
+  user,
+  choseTodo,
+}) => {
+  const status = choseTodo.completed ? (
+    <strong className="has-text-success">Done</strong>
+  ) : (
+    <strong className="has-text-danger">Planned</strong>
+  );
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
-
-      {true ? (
+      {loading ? (
```

Comment:
looks like problem with loader here. you don't set to false loading when close modal.
but in general loading from app shouldn't come to modal. 
in modal should be loading for user.
also app shouldn't know anything about user. load user in modal.